tutorials/plot_background_statistics.py

```diff
@@ -246,29 +255,29 @@ def plot_t_p(t, p, title, mcc, axes=None):
 fig, ax = plt.subplots(figsize=(4, 3))
 ax.scatter(N, p_type_I, 3)
 ax.set(xlim=N[[0, -1]], ylim=[0, 1], xlabel='$N_{\mathrm{test}}$',
-       ylabel=u'Probability of ≥ 1\ntype I error')
+       ylabel=u'Probability of at least\none type I error')
 ax.grid(True)
 fig.tight_layout()
-mne.viz.utils.plt_show()
+fig.show()
 
 ###############################################################################
-# To combat this problem, multiple methods exist. Typically these
-# provide control over either the:
+# To combat this problem, several methods exist. Typically these
+# provide control over either one of the following two measures:
 #
-# 1. `Familywise error rate (FWER) <fwer>`_
+# 1. `Familywise error rate (FWER) <https://en.wikipedia.org/wiki/Family-wise_error_rate>`_
```

Comment:
PEP8 line length, should probably be reverted

https://travis-ci.org/mne-tools/mne-python/jobs/400807122#L3216